config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- name: sig-auth-serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+    testgrid-tab-name: upgrade-tests
+    # TODO: use sig-auth mailing list
+    testgrid-alert-email: zshihang@google.com
+    testgrid-num-failures-to-alert: "3"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  # TODO: lower to once a day
+  interval: 2h
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-d8fda77-master
+        args:
+        - --timeout=920
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
```

Comment:
I missed this... these are bootstrap.py parameters, they shouldn't be here if `decorate: true` is being used.

It's more preferable to use `decorate: true` than to revert to bootstrap.py, as bootstrap has been deprecated for years

Reply:
See https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml#L73-L82 for an example of what `command:` and `args:` should be instead

Reply:
done. thank you for the review.